src/gui/appclient.cpp

```diff
@@ -157,7 +157,7 @@ AppClient::AppClient(int &argc, char **argv)
 
     if (count == CHECKCOMMSTATUS_TRIALS) {
         QMessageBox msgBox;
-        msgBox.setText(tr("Unable to communicate with application server!"));
+        msgBox.setText(tr("The application server did not respond on time, the client will be restarted."));
```

Comment:
Suggestion: "The application server did not respond on time: the client will be restarted."

Reply:
Ok, done